hedera-node/src/test/java/com/hedera/services/store/contracts/precompile/SyntheticTxnFactoryTest.java

```diff
@@ -356,15 +360,31 @@ void createsExpectedNodeStakeUpdate() {
                                 .setStake(987_654_321L)
                                 .setStakeRewarded(54_321L)
                                 .build());
+        propertySource.ensureProps();
+        dynamicProperties =
+                new GlobalDynamicProperties(new HederaNumbers(propertySource), propertySource);
+        subject = new SyntheticTxnFactory(dynamicProperties);
 
-        final var txnBody = subject.nodeStakeUpdate(timestamp, nodeStakes);
+        final var txnBody = subject.nodeStakeUpdate(timestamp, nodeStakes, propertySource);
 
         assertTrue(txnBody.hasNodeStakeUpdate());
         assertEquals(timestamp, txnBody.getNodeStakeUpdate().getEndOfStakingPeriod());
         assertEquals(123_456_789L, txnBody.getNodeStakeUpdate().getNodeStake(0).getStake());
         assertEquals(1_234_567L, txnBody.getNodeStakeUpdate().getNodeStake(0).getStakeRewarded());
         assertEquals(987_654_321L, txnBody.getNodeStakeUpdate().getNodeStake(1).getStake());
         assertEquals(54_321L, txnBody.getNodeStakeUpdate().getNodeStake(1).getStakeRewarded());
+        assertEquals(17_808L, txnBody.getNodeStakeUpdate().getMaxStakingRewardRatePerHbar());
+        assertEquals(0L, txnBody.getNodeStakeUpdate().getNodeRewardFeeFraction().getNumerator());
+        assertEquals(
+                100L, txnBody.getNodeStakeUpdate().getNodeRewardFeeFraction().getDenominator());
+        assertEquals(365, txnBody.getNodeStakeUpdate().getStakingPeriodsStored());
+        assertEquals(1L, txnBody.getNodeStakeUpdate().getStakingPeriod());
```

Comment:
Is this just because of different test data or we accidentally sending the unit as days instead of minutes?

```suggestion
        assertEquals(1440L, txnBody.getNodeStakeUpdate().getStakingPeriod());
```